CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Include lockfile paths when analyzing projects
 - Generate and use API Keys instead of OpenID Connect tokens
+- Use relative paths for project lockfile path submissions
```

Comment:
There is no need for a changelog entry in the PR. These changes are fully covered by the changelog entry above (which is still unreleased):

```
- Include lockfile paths when analyzing projects
```

So you can remove this line

```suggestion
```